### PR TITLE
Configuring wazuh-api from puppet

### PR DESCRIPTION
**Pull request by rafaelfc-olx**
Following up on https://github.com/wazuh/wazuh-puppet/pull/31, here's a patch to manage the basic items of the wazuh-api configuration from puppet.

### CODE REVIEW

**Comment by jnarvaezp** [comment]
Hi @rafaelfc-olx thanks for your contribution, let me review and merge!

File: manifests/server.pp [hunk 1]
```diff
@@ -180,6 +182,19 @@
...
     }
 
+    # wazuh-api config.js
+    # this hash is currently only covering the basic config section of config.js
+    # TODO: allow customization of the entire config.js
+    # for reference: https://documentation.wazuh.com/current/user-manual/api/configuration.html
+    file { '/var/ossec/api/configuration/config.js':
+      content => template($api_config_template),
+      owner   => 'root',
+      group   => 'root',
```
Comment (jnarvaezp) [inline manifests/server.pp]: Hi @rafaelfc-olx 

The right group is ossec

JN

**Comment by jnarvaezp** [comment]
Hi @rafaelfc-olx 

I have left a comment about the group assignment, the rest of the code I have revised it and it works, if you prefer you can make the change, or can I accept the pullrequest and I make the change, which option do you think is better?

File: manifests/server.pp (hunk 1 above)
Comment (rafaelfc-olx) [reply manifests/server.pp]: just changed, thanks again for the heads up :)

**Comment by jnarvaezp** [comment]
HI @rafaelfc-olx 

I've tested the changes, and they work properly.

Thank you for your contribution.

**Comment by jlruizmlg** [comment]
Hi @rafaelfc-olx, first at all thanks for all your contributions, have the API in puppet is amazing, 

We think that can be more easy to use if we could use a different class to install the API, @jnarvaezp has created a new class in the next pull request: https://github.com/wazuh/wazuh-puppet/pull/39 

What do you think?

**Pull request merged**